pkg/kubecost/asset.go

```diff
@@ -3023,7 +3025,7 @@ func (asr *AssetSetRange) Length() int {
 
 func (asr *AssetSetRange) MarshalJSON() ([]byte, error) {
 	asr.RLock()
-	asr.RUnlock()
+	defer asr.RUnlock()
```

Comment:
👀

Reply:
I know. Damn.

Reply:
Just been locking and unlocking for kicks.